setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
@david-caro see my past comment https://github.com/inveniosoftware/invenio-workflows/pull/27#discussion_r72216443 and point me to the place where you changed it.